### DISCUSSION TITLE
Docker arch

### DISCUSSION THREAD

**Discussion by sausix** [mat/besticon#85] [General]
Why isn't here an issue tab? Issue reports not allowed?

The new docker images are tagged as `linux/arm64/v8` which throws warnings. Previous versions have been labeled `linux/amd64`.  

Seems still to be amd64 by viewing the Dockerfile: `COPY /app/bin/linux_amd64/iconserver /iconserver # buildkit`

**Comment by mat** [comment]
I believe [this](https://github.com/mat/besticon/commit/9da96158998f5441eeb4a7ae068696ef6d52fbe5) should fix it. Can you `docker pull` the latest image and confirm?
